File: 백준/Bronze/10810. 공 넣기/공 넣기.js
const fs = require('fs');
const input = fs.readFileSync(0, 'utf-8').trim().split('\n');

const [N, M] = input[0].split(" ").map(Number);
let basket = new Array(N).fill("0");

for (let n = 0; n < M; n++) {
  const [i, j, k] = input[n + 1].split(" ");

  for (let target = parseInt(i) - 1; target < j; target++) {
    basket[target] = k;
  }
}

console.log(basket.join(" "));

File: 백준/Bronze/10810. 공 넣기/공 넣기.swift
let cond: [Int] = readLine()!.split(separator: " ").map { Int($0)! }
let (n, m): (Int, Int) = (cond[0], cond[1])
var basket: [String] = Array(repeating: "0", count: n)

for _ in 0 ..< m {
  let input: [Int] = readLine()!.split(separator: " ").map { Int($0)! }
  for i in input[0] ... input[1] {
    basket[i - 1] = String(input[2])
  }
}
print(basket.joined(separator: " "))